README.docker.md

```diff
@@ -0,0 +1,141 @@
+This document describes our development docker configuration.
+
+> Instruction below are written for Debian linux. For other distros/OSes it can require some changes
+
+We assume that application itself will be run in host system natively, docker is used to only host services used by app:
```

Comment:
In production, we use a separae AWS RDS service for the main production database, and not a Mysql instance on localhost.

Reply:
@damisul ?